packages/insomnia/src/ui/components/markdown-editor.tsx

```diff
@@ -18,85 +16,59 @@ interface Props {
   tall?: boolean;
 }
 
-interface State {
-  markdown: string;
-}
-
-@autoBindMethodsForReact(AUTOBIND_CFG)
-export class MarkdownEditor extends PureComponent<Props, State> {
-  _editor: UnconnectedCodeEditor | null = null;
-
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      markdown: props.defaultValue,
-    };
-  }
+const MarkdownEditorForwarded: ForwardRefRenderFunction<UnconnectedCodeEditor, Props> = ({
+  mode,
+  placeholder,
+  defaultPreviewMode,
+  className,
+  tall,
+  defaultValue,
+  onChange,
+}, ref): ReactElement => {
```

Comment:
```suggestion
export const MarkdownEditor: forwardRef<UnconnectedCodeEditor, Props> = ({
  mode,
  placeholder,
  defaultPreviewMode,
  className,
  tall,
  defaultValue,
  onChange,
}, ref) => {
```
this can also work then you don't need line 74
```
export const MarkdownEditor = forwardRef(MarkdownEditorForwarded);
```

Reply:
done